File: sdk/async-scala/src/main/scala/io/swagger/client/model/CreditCard.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class CreditCard (
  ID: Option[String],
Token: Option[String],
CardType: Option[String],
PartialAccountNumber: Option[String],
CardholderName: Option[String],
ExpirationDate: Option[Date],
xp: Option[Any]
)

File: sdk/javascript-closure-angular/API/Client/CreditCard.js
goog.provide('API.Client.CreditCard');

/**
 * @record
 */
API.Client.CreditCard = function() {}

/**
 * @type {!string}
 * @export
 */
API.Client.CreditCard.prototype.ID;

/**
 * @type {!string}
 * @export
 */
API.Client.CreditCard.prototype.token;

/**
 * @type {!string}
 * @export
 */
API.Client.CreditCard.prototype.cardType;

/**
 * @type {!string}
 * @export
 */
API.Client.CreditCard.prototype.partialAccountNumber;

/**
 * @type {!string}
 * @export
 */
API.Client.CreditCard.prototype.cardholderName;

/**
 * @type {!API.Client.date}
 * @export
 */
API.Client.CreditCard.prototype.expirationDate;

/**
 * @type {!API.Client.Object}
 * @export
 */
API.Client.CreditCard.prototype.xp;

